src/Surfnet/ServiceProviderDashboard/Application/Service/EntityService.php

```diff
@@ -143,6 +146,14 @@ public function getEntityByIdAndTarget($id, $manageTarget, Service $service)
                 $entity = $this->queryRepositoryProvider
                     ->getManageTestQueryClient()
                     ->findByManageId($id);
+
+                // Allow actions on Resource Servers (viewing them outside of our team)
+                if ($entity->getProtocol()->getProtocol() !== Constants::TYPE_OPENID_CONNECT_TNG_RESOURCE_SERVER) {
```

Comment:
not related to this PR, but `->getProtocol()->getProtocol()` seems like a bad name for either the first or the second?

Reply:
Yup, that an eyesore for me last couple of months (years even). It may be updated some time soon, but certainly not in this PR